features/step_definitions/journey_test_steps.rb

```diff
@@ -2052,3 +2057,21 @@ def check_latest_row_correct(service_status)
 Then /^I am on the public view of the opportunity$/ do
   visit("#{dm_frontend_domain}/digital-outcomes-and-specialists/opportunities/#{@published_brief["id"]}")
 end
+
+And /^The count of unanswered questions remaining for the buyer requirement is correctly shown$/ do
+  if store.table_rows_with_optional_count >= 1 or store.required_question_count >= 1
+    if store.table_rows_with_optional_count >= 1
+      page.find(:xpath, "//a[contains(@href, '#{store.current_brief_id}')]/../../../td[3]/span[text()]").should have_content("#{store.table_rows_with_optional_count}"' optional')
+    end
+
+    if store.required_question_count >= 1
+      page.find(:xpath, "//a[contains(@href, '#{store.current_brief_id}')]/../../../td[3]/span[text()]").should have_content("#{store.required_question_count}"' required')
+    end
+  else
+    page.should have_no_selector(:xpath, "//a[contains(@href, '#{store.current_brief_id}')]/../../../td[3]/span[text()]")
+  end
+end
+
+Given /^I navigate directly to the '(.*)' dasboard page$/ do |user_type|
+  step "I navigate directly to the page \'#{user_type}\'"
```

Comment:
Why do we need this as a separate step? Why can't we just use `Given I navigate directly to the page 'buyers'`?